packages/vuetify/src/components/VTextField/VTextField.sass

```
@@ -24,12 +24,21 @@
     .v-field
       cursor: text
 
-    .v-field__input
-      @at-root #{selector.append('.v-text-field--prefixed', &)}
-        --v-field-padding-start: #{$text-field-input-padding-start}
+      &:not(.v-field--reverse)
+        .v-field__input
+          @at-root #{selector.append('.v-text-field--prefixed', &)}
+            --v-field-padding-start: #{$text-field-input-padding-start}
 
-      @at-root #{selector.append('.v-text-field--suffixed', &)}
-        --v-field-padding-end: #{$text-field-input-padding-end}
+          @at-root #{selector.append('.v-text-field--suffixed', &)}
+            --v-field-padding-end: #{$text-field-input-padding-end}
+
+      &:is(.v-field--reverse)
```

This is redundant with only one selector in it
```suggestion
      &.v-field--reverse
```